README.md

```diff
@@ -8,47 +8,106 @@ Rust-Lightning
 Rust-Lightning is a Bitcoin Lightning library written in Rust. The main crate,
 `lightning`, does not handle networking, persistence, or any other I/O. Thus,
 it is runtime-agnostic, but users must implement basic networking logic, chain
-interactions, and disk storage.
+interactions, and disk storage. More information is available in the `About`
+section.
 
 The `lightning-net-tokio` crate implements Lightning networking using the
 [Tokio](https://github.com/tokio-rs/tokio) async runtime.
 
+The `lightning-persister` crate implements persistence for channel data that
+is crucial to avoiding loss of channel funds. Sample modules for persistence of
+other Rust-Lightning data is coming soon.
+
 Status
 ------
 
 The project implements all of the BOLT specifications in the 1.0 spec except
 for [channel queries](https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#query-messages). The
 implementation has pretty good test coverage that is expected to continue to
-improve. There are a number of internal refactorings being done now that will
-make the code base more welcoming to new contributors. It is also anticipated
-that as developers begin using the API, the lessons from that will result in
-changes to the API, so any developer using this API at this stage should be prepared
-to embrace that. The current state is sufficient for a developer or project to
-experiment with it. Recent increased contribution rate to the project is expected
-to lead to a high quality, stable, production-worthy implementation in 2020.
+improve. It is also anticipated that as developers begin using the API, the
+lessons from that will result in changes to the API, so any developer using this
+API at this stage should be prepared to embrace that. The current state is
+sufficient for a developer or project to experiment with it. Recent increased
+contribution rate to the project is expected to lead to a high quality, stable,
+production-worthy implementation in 2021.
 
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+About
+-----------
+LDK/Rust-Lightning is a generic library which allows you to build a lightning
+node without needing to worry about getting all of the lightning state machine
+and on-chain punishment code exactly correct. Note that Rust-Lightning isn't, in
```

Comment:
I would switch to "on-chain reaction", really on-chain stuff is far wider than punishment. Also precise routing and HTLC forwarding. E.g Electrum doesn't implement forwarding IIRC, so a good point to mention.

Reply:
Isn't forwarding implied when we say we're fully BOLT-compliant? Added routing though and amended the "on-chain" language.

Reply:
I don't think "BOLT-compliant" is well-defined - I'd call a client that doesn't do forwarding compliant.

Reply:
Agree, forwarding/routing are what make Lightning actually a network, without it's just nice bidirectional payment channel.